planutils/packages/lama/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+downward --alias lama-first $@
```

Comment:
If this is running `lama-first`, shouldn't the package also be called `lama-first`. From the name `lama`, I'd expect it to run the anytime variant (ideally `lama` but the legacy IPC verison `seq-sat-lama-2011` would also be an option).